### PR TITLE
Feat/team

### PR DESCRIPTION
**Pull request by MatiasOS**
### Components
- Team section

@skkr I need help with styles here. I think the best is to close this way and later you can fix when you have free time.

### CODE REVIEW

**Comment by skkr** [comment]
@menschikov that's not the style we have. Please let's handle all design changes as requirements in a future iteration as disused. 

I can work on this later.

**Comment by menschikov** [comment]
@skkr that was a question. Is that a final version of section? If yes, it looks incomplete

**Comment by skkr** [comment]
@menschikov right, sorry! :)

No, it should look like the current design. So text alignment is wrong, and the grid is wrong too.

@MatiasOS it should be easy to fix, ping me if need help.

File: src/features/Foundation/Team/Member.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  constructor(props: TeamMemberType) {
+    super(props);
+    this.state = {
+      isOpened: false,
+    };
+  }
+
+  handleCollapseClick = () => {
+    const { isOpened } = this.state;
+    this.setState({ isOpened: !isOpened });
```
[the remark this answers is not in the document]
Comment (MatiasOS) [reply src/features/Foundation/Team/Member.jsx]: Changed

**Pull request merged**